app/javascript/channels/meetle_channel.js

```diff
@@ -3,15 +3,21 @@ import { updateMap } from "../packs/map";
 
 const initMeetleCable = () => {
   const locationsContainer = document.getElementById('location');
+  // select all the upvote number in the html
   if (locationsContainer) {
     const id = locationsContainer.dataset.meetleId;
 
     consumer.subscriptions.create({ channel: "MeetleChannel", id: id }, {
       received(data) {
-        console.log(data);
-        locationsContainer.innerHTML = data.partial;
-        updateMap(data.coordinates);
-        // initMapbox;
+        // do that if uvote not there
+        if (data.upvote) {
+          Object.keys(data.upvote).forEach((id) => {
+            document.querySelector(`#card-result-${id} .topright`).innerHTML = `<i class="fas fa-heart"></i> ${data.upvote[id]}`;
```

Comment:
this icon is broadcasted with the number of the upvote, which is not what i want but i tried other methods and it sucked even more so...